tests/fxl-spread_landscape/OPS/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>If rendition:spread is "landscape", synthetic spreads should be rendered when the reading system is in landscape orientation.</dc:description>
```

Comment:
"should be rendered only when" ("only" is in the spec and is important). Or make that a separate sentence: "In portrait mode..."